yolort/utils/update_module_state.py

```diff
@@ -12,6 +12,8 @@ def update_module_state_from_ultralytics(
     arch: str = 'yolov5s',
     version: str = 'v4.0',
     num_classes: int = 80,
+    pretrained_file: str = None,
```

Comment:
Hi, I guess that the type annotation should be `Optional[str]` here.

```suggestion
    pretrained_file: Optional[str] = None,
```